.travis.d/deploy_docs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+cd $TRAVIS_BUILD_DIR
+git fetch origin gh-pages:origin/gh-pages
+git checkout origin/gh-pages -b gh-pages || git checkout --orphan gh-pages
```

Comment:
it's complicate. actually we use github environ. travis also join this system.
but, in `git` environ,  `fork repo` or `clone remote repo` not mean copied all branches.

so, in my cases, i want not receiving fail this cases, it's just repo problem, not a branches problem.
i guess, problem is travis shown build failure as broken deploy, it's false alarm.

anyway, really `we` want this, i'll fix them. :/